Board and mailing functionality

angelwalia3 opened the pull request:
Set up the postits board and send mail functionality. Add project id to the pom file to aid deployment. Deployed site : https://awalia-sps-summer20.et.r.appspot.com/

angelwalia3 committed:
add whiteboard

angelwalia3 committed:
add leave a message text box and send button

angelwalia3 committed:
add post-its randomly on the board, positions not correctly determined

angelwalia3 committed:
Correct the random number generation to have the postits appear in side the board

angelwalia3 committed:
add text to postit

angelwalia3 committed:
set up mailing functionality and fix header links and add the license
text

angelwalia3 committed:
set up mailing functionality and fix header links

angelwalia3 committed:
remove projects link

angelwalia3 committed:
remove projects link

angelwalia3 committed:
add sub msg for the text label and add project name in pom file

angelwalia3 committed:
add sub msg for the text label and add project name in pom file

streetdogg commented on portfolio/src/main/webapp/style.css:
```diff
@@ -137,4 +137,80 @@ nav ul li a{
+	height: auto;
+	width: 10%;
+	display: none;
+	/*top: 25%;*/
+	/*left: 30%;*/
+	/*top: 62%;
+	left: 62%;*/
```
Should this be removed? Looks like this is commented.

angelwalia3 replied:
Yes, this is commented out but I left it there on purpose. It is some information regarding the proper placement of the post-its on the board. Hence I thought of leaving it there for future reference.

streetdogg replied:
then can we add a comment on top stating these are left here on purpose? That way this will not be confused with dead/unused code...

angelwalia3 replied:
okay noted!

streetdogg commented on portfolio/src/main/webapp/script.js:
```diff
@@ -1,28 +1,51 @@
+function sendEmail() {
+	Email.send({
+	Host: "smtp.gmail.com",
+	Username : "spsportfolio03@gmail.com",
+	Password : "catchrandommails",
+	To : "spsportfolio03@gmail.com",
+	From : "spsportfolio03@gmail.com",
+	Subject : "Message from your protfolio",
+	Body : document.getElementById("msg").value,
```
so this mails the content to self. cool!

streetdogg commented on portfolio/src/main/webapp/script.js:
```diff
@@ -1,28 +1,51 @@
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+var turn = 0;
```
please add back the license header that got deleted from here. :D

streetdogg reviewed (commented):
Nit: The license text in `portfolio/src/main/webapp/script.js` seems to be deleted. Please add that back.

angelwalia3 committed:
add license text
Merge branch 'AngelDev' of https://github.com/angelwalia3/MySPSPortfolio into AngelDev

angelwalia3 committed:
add license text

streetdogg commented on portfolio/src/main/webapp/script.js:
```diff
@@ -12,17 +12,54 @@
+	return [top,left];
+}
+
+function sendEmail() {
```
should the space between `)` and `{` be removed? Just for consistency across all functions?
(Sorry for not catching this earlier...)

angelwalia3 replied:
edited!


> should the space between `)` and `{` be removed? Just for consistency across all functions?
> (Sorry for not catching this earlier...)

yup edited!

angelwalia3 committed:
Add license and remove extra space

angelwalia3 committed:
Merge branch 'AngelDev' of https://github.com/angelwalia3/MySPSPortfolio into AngelDev

streetdogg reviewed (approved):
Looks good to me.
